### PR TITLE
Recent papers by Daan

### PR DESCRIPTION
**Pull request by dhil**
I have added two recent papers by Daan.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,15 @@ computational effects.
 
 ### 2017
 
+* **Structured asynchrony with algebraic effects** (TyDe 2017)  
+  by Daan Leijen  
+  ([pdf](https://www.microsoft.com/en-us/research/wp-content/uploads/2017/05/asynceffects-msr-tr-2017-21.pdf))
```
Comment (yallop) [inline README.md]: This one's a Microsoft TR, not a TyDe paper.  Could you please fix either the description or the URL?

File: README.md [hunk 2]
```diff
@@ -49,6 +49,15 @@ computational effects.
...
+
+* **Implementing Algebraic Effects in C** (APLAS 2017)  
+  by Daan Leijen  
+  ([pdf](https://www.microsoft.com/en-us/research/wp-content/uploads/2017/06/algeff-in-c-tr-v2.pdf))
```
Comment (yallop) [inline README.md]: This one's another TR, not an APLAS paper.

File: README.md (hunk 1 above)
Comment (dhil) [reply README.md]: May I link to OpenTOC? http://www.sigplan.org/OpenTOC/tyde17.html

File: README.md (hunk 2 above)
Comment (dhil) [reply README.md]: Right. It is however accepted https://www-aplas.github.io/program.html

File: README.md (hunk 1 above)
Comment (yallop) [reply README.md]: Yes, but please label accordingly, e.g. `([OpenTOC redirection](…))` rather than `([pdf](…))`

File: README.md (hunk 2 above)
Comment (yallop) [reply README.md]: Glad to hear it!  If you change the label to `([Tech Report](…))` I'm happy to merge.

**Comment by yallop** [comment]
Thanks!

**Pull request merged**